src/libs/OptionsListUtils.js

```diff
@@ -599,6 +599,32 @@ function isCurrentUser(userDetails) {
     return _.some(_.keys(loginList), (login) => login.toLowerCase() === userDetailsLogin.toLowerCase());
 }
 
+/**
+ * Calculates count of all enabled options
+ *
+ * @param {Object[]} options - an initial strings array
+ * @param {Boolean} options[].enabled - a flag to enable/disable option in a list
+ * @param {String} options[].name - a name of an option
+ * @returns {Boolean}
+ */
+function getEnabledCategoriesCount(options) {
+    return _.chain(options)
+        .filter((option) => Boolean(option.enabled))
+        .value().length;
+}
+
+/**
+ * Verifies that there is at least one enabled option
+ *
+ * @param {Object[]} options - an initial strings array
+ * @param {Boolean} options[].enabled - a flag to enable/disable option in a list
+ * @param {String} options[].name - a name of an option
+ * @returns {Boolean}
+ */
+function hasEnabledOptions(options) {
+    return _.some(options, (option) => Boolean(option.enabled));
```

Comment:
can we use:
```
    return options.some((option) => option.enabled);
```
no loadash, just builtin solutions

Reply:
However, I agree to simplify the logic. Let me try to play with it.

Reply:
Done - 4f875ea1641b11fa676b5bafbd3c0f3facd2d1fc.